nautobot/docs/installation/ubuntu.md

```diff
@@ -112,7 +112,7 @@ owners.
 
 Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
 
-mysql> CREATE DATABASE nautobot CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;
+mysql> CREATE DATABASE nautobot;
```

Comment:
Same question here - has this been checked on ubuntu specifically?

Reply:
Yep!